pkg/middlewares/auth/forward.go

```diff
@@ -45,13 +45,23 @@ func NewForward(ctx context.Context, next http.Handler, config dynamic.ForwardAu
 		trustForwardHeader:  config.TrustForwardHeader,
 	}
 
+	// Ensure our request client does not follow redirects
+	fa.client = http.Client{
+		CheckRedirect: func(r *http.Request, via []*http.Request) error {
+			return http.ErrUseLastResponse
+		},
+		Timeout: 30 * time.Second,
```

Comment:
Is this a sane default to hard code in?

Or should we allow this to be tuneable for large environments?

Reply:
for now we don't need to allow a custom value.